src/lib.rs

```diff
@@ -885,51 +925,747 @@ impl GooseAttack {
+                // Do not default to status_codes on Worker.
+                if self.attack_mode == GooseMode::Worker {
+                    false
+                } else {
+                    default
+                }
+            } else {
+                false
+            };
+        }
+    }
+
+    // Determine if the only_summary flag is enabled.
+    fn set_only_summary(&mut self) {
+        // Overload self.configuration.only_summary so it's available on Worker.
+        if !self.configuration.only_summary {
+            self.configuration.only_summary = if let Some(default) = self.defaults.only_summary {
+                // Do not default to only_summary on Worker.
+                if self.attack_mode == GooseMode::Worker {
+                    false
+                } else {
+                    default
+                }
+            } else {
+                false
+            };
+        }
+    }
+
+    // Determine if the no_task_metrics flag is enabled.
+    fn set_no_task_metrics(&mut self) {
+        // Overload self.configuration.no_task_metrics so it's available to Users.
+        if !self.configuration.no_task_metrics {
+            self.configuration.no_task_metrics =
+                if let Some(default) = self.defaults.no_task_metrics {
+                    // Do not default to no_task_metrics on Worker.
+                    if self.attack_mode == GooseMode::Worker {
+                        false
+                    } else {
+                        default
+                    }
+                } else {
+                    false
+                };
+        }
+    }
+
+    // Determine if the no_metrics flag is enabled.
+    fn set_no_metrics(&mut self) {
+        // Overload self.configuration.no_metrics so it's available to Users.
+        if !self.configuration.no_metrics {
+            self.configuration.no_metrics = if let Some(default) = self.defaults.no_metrics {
+                // Do not default to no_metrics on Worker.
+                if self.attack_mode == GooseMode::Worker {
+                    false
+                } else {
+                    default
+                }
+            } else {
+                false
+            };
+        }
+    }
+
+    // Determine if the sticky_follow flag is enabled.
+    fn set_sticky_follow(&mut self) {
+        // Overload self.configuration.sticky_follow so it's available to Users.
+        if !self.configuration.sticky_follow {
+            self.configuration.sticky_follow = if let Some(default) = self.defaults.sticky_follow {
+                // Do not default to sticky_follow on Worker.
+                if self.attack_mode == GooseMode::Worker {
+                    false
+                } else {
+                    default
+                }
+            } else {
+                false
+            };
+        }
+    }
+
+    #[cfg(feature = "gaggle")]
+    // Determine if no_hash_check is enabled.
+    fn no_hash_check(&self) -> bool {
```

Comment:
@LionsAd Should this be `get_no_hash_check` or `calculate_no_hash_check`? Or should we just overload `configuration` consistently?

Reply:
updated to `set_no_hash_check()` which updates `configuration` with default if set